src/transformers/models/audio_spectrogram_transformer/feature_extraction_audio_spectrogram_transformer.py

```diff
@@ -218,6 +233,8 @@ def __call__(
         # extract fbank features and pad/truncate to max_length
         features = [self._extract_fbank_features(waveform, max_length=self.max_length) for waveform in raw_speech]
 
+        features = [self.mask(feature, freqm, timem) for feature in features]
```

Comment:
Let's apply it only if `freqm` and `timem` are not None, rather than checking that in the `self.mask` method.